wallet/src/de/schildbach/wallet/ui/transactions/TransactionResultViewBinder.kt

```diff
@@ -221,7 +215,7 @@ class TransactionResultViewBinder(
                 transactionTitle.setTextColor(ContextCompat.getColor(ctx, R.color.dash_blue))
                 transactionTitle.text = ctx.getText(R.string.transaction_details_amount_sent)
                 transactionAmountSignal.text = "-"
-                transactionAmountSignal.isVisible = !TransactionUtils.isEntirelySelf(tx, wallet)
+                transactionAmountSignal.isVisible = true
```

Comment:
The fact that we want to include the fee into the total amount kinda clashes with our idea of removing the signal from internal transactions (it'll look as if we had received the fee amount).
For now, let's just keep it the way it is in production currently.